core/src/main/java/ai/timefold/solver/core/impl/move/streams/maybeapi/generic/Moves.java

```diff
@@ -0,0 +1,96 @@
+package ai.timefold.solver.core.impl.move.streams.maybeapi.generic;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import ai.timefold.solver.core.preview.api.domain.metamodel.PlanningListVariableMetaModel;
+import ai.timefold.solver.core.preview.api.domain.metamodel.PlanningVariableMetaModel;
+import ai.timefold.solver.core.preview.api.domain.metamodel.PositionInList;
+import ai.timefold.solver.core.preview.api.move.Move;
+
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
+@NullMarked
+public final class Moves {
```

Comment:
The name seems incomplete. Perhaps we should call it `MoveOperation`,  `MoveUtil`, etc.

Reply:
This class is very helpful. I sometimes get confused about importing the correct move class.

Reply:
Short doesn't mean incomplete. :-)

We have Joiners, ConstraintCollectors.
JDK has Collections, Files, ...

IMO this name is good.